storage/replica.go

```diff
@@ -1183,21 +1185,21 @@ func (r *Replica) handleRaftReady() error {
 
 func (r *Replica) sendRaftMessage(msg raftpb.Message) {
 	groupID := r.RangeID
+
 	r.store.mu.Lock()
-	toReplica, err := r.store.replicaDescriptorLocked(groupID, roachpb.ReplicaID(msg.To))
-	if err != nil {
-		log.Warningf("failed to lookup recipient replica %d in group %s: %s", msg.To, groupID, err)
-		r.store.mu.Unlock()
+	toReplica, toErr := r.store.replicaDescriptorLocked(groupID, roachpb.ReplicaID(msg.To))
```

Comment:
this looks like a change in behaviour in that it will now look up both descriptors even if one fails. is that intended?